src/pipecat/services/deepgram.py

```diff
@@ -129,15 +134,19 @@ def __init__(
             punctuate=True,
             profanity_filter=True,
             vad_events=False,
-        ),
-        **kwargs,
-    ):
-        super().__init__(**kwargs)
+        )
 
-        self._settings = vars(live_options)
+        merged_options = default_options
+        if live_options:
+            merged_options = LiveOptions(**{**default_options.to_dict(), **live_options.to_dict()})
+        self._settings = merged_options
```

Comment:
In running this locally, I see that _update_settings expects _settings to be a Dict.
```suggestion
        self._settings = merged_options.to_dict()
```